README.md

```diff
@@ -314,6 +322,8 @@ For example, if your environment installs its javascript npm dependencies at the
 
 **NB: This is experimental functionality. Instructions and syntax for configuration, etc. are currently in flux.**
 
+See also the [demo/treebank-translation.html](treebank-translation.html) file in the demos directory.
+
```

Comment:
I have tested http://localhost:8888/demo/treebank-translation.html
`nova` has a treebank diagram
but `transformed` has not

![image](https://user-images.githubusercontent.com/12377640/65647409-f803b780-e041-11e9-9a66-cf150ab2ba50.png)

Reply:
right -- only the latin text is treebanked.